packages/tokamak/core-utils/src/optimism/constants.ts

```diff
@@ -0,0 +1,41 @@
+import { ethers } from 'ethers'
+
+/**
+ * Predeploys are Solidity contracts that are injected into the initial L2 state and provide
+ * various useful functions.
+ * Notes:
+ * 0x42...04 was the address of the OVM_ProxySequencerEntrypoint. This contract is no longer in
+ * use and has therefore been removed. We may place a new predeployed contract at this address
+ * in the future. See https://github.com/ethereum-optimism/optimism/pull/549 for more info.
+ */
+export const predeploys = {
+  L2ToL1MessagePasser: '0x4200000000000000000000000000000000000016',
+  DeployerWhitelist: '0x4200000000000000000000000000000000000002',
+  L2CrossDomainMessenger: '0x4200000000000000000000000000000000000007',
+  GasPriceOracle: '0x420000000000000000000000000000000000000F',
+  L2StandardBridge: '0x4200000000000000000000000000000000000010',
+  SequencerFeeVault: '0x4200000000000000000000000000000000000011',
+  OptimismMintableERC20Factory: '0x4200000000000000000000000000000000000012',
+  L1BlockNumber: '0x4200000000000000000000000000000000000013',
+  L1Block: '0x4200000000000000000000000000000000000015',
+  LegacyERC20ETH: '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000',
+  WETH9: '0x4200000000000000000000000000000000000006',
```

Comment:
I think we should update WETH9 -> WNativeToken, and add ETH to predeploy

Reply:
Oh, the target branch is main, so it is OK

Reply:
It's correct! Thank you, Brave! I will do it on my next ticket!

Reply:
I will merge main in OR-1257(your branch). So I think I can work for it at that time as well!